mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/MongoDBDialect.java

```diff
@@ -1699,6 +1697,7 @@ private static BatchInsertionTask getOrCreateBatchInsertionTask(Map<MongoCollect
 	}
 
 	private static void flushInserts(Map<MongoCollection<Document>, BatchInsertionTask> inserts) {
+		//@todo move GridFS features here
```

Comment:
Please, remove the todo from the PR as they will end in the log history or on master upstream and we prefer to avoid this.